test/hotspot/jtreg/compiler/macronodes/TestEliminationOfAllocationWithoutUse.java

```diff
@@ -0,0 +1,338 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is the copyright year accurate?

Reply:
It's your test that took over and finished so you tell me: do you want the copyright updated?

Reply:
Ah right. Well :see_no_evil:  I guess you could write `2024, 2025` then :)